runtimes/peregrine/src/lib.rs

```diff
@@ -90,7 +90,7 @@ impl_opaque_keys! {
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("mashnet-node"),
-	impl_name: create_runtime_str!("mashnet-node"),
+	impl_name: create_runtime_str!("kilt-peregrine"),
```

Comment:
Not sure I like this... EIther we don't change it at all, or we change both. What's the reason for changing only the implementation?

Reply:
The idea was that there is at least one difference in the naming. spec name might be the same but at least the impl name is different.

Reply:
I think spec name should also be different. That's the assumption the type definitions do as well. I think impl name is probably a different implementation of the Peregrine runtime. But the spec name indicates the runtime. If Peregrine is not mashnet, then the spec name should not be mashnet either.

Reply:
Yes the only reason I'm not changing the spec name is, that I'm not 100% sure about the consequences

Reply:
Ok let's do this as part of another PR once we researched all the consequences of renaming the spec.

[8adcaa4](https://github.com/KILTprotocol/kilt-node/pull/554/commits/8adcaa454c4ba8fe4dc2de08634695d67d4c4861)